values.yaml

```diff
@@ -1,21 +1,49 @@
 # Global settings for the platform
 global:
-  # Set this to 'true' to deploy Kafka/Apicurio inside this chart
+  # Include basic infrastructure setup (mainly for local development)
   includeKafka: true
   includeApicurio: true
   includeApiGateway: true
+  includeStatusDb: true
+  includeS3: true
 
 # Configuration for the internal Dev infrastructure
 infra:
+  # KAFKA CONFIG
   kafka:
     image: quay.io/strimzi/kafka:latest-kafka-3.7.0
     storageClass: "standard"
+  # APICURIO CONFIG
   apicurio:
     image: quay.io/apicurio/apicurio-registry:latest
   gateway:
     route:
       enabled: false
       host: ""
+  # POSTGRESQL CONFIG (Status DB)
+  statusDb:
+    image: quay.io/fedora/postgresql-16:latest
+    env:
+      database: sbomer_status
```

Comment:
I think its nice to allow overrides like this but they should be backed by secrets and in the case these values are omitted they are omitted we should randomly generate username and passwords in secrets.